include/zephyr/bluetooth/classic/hfp_hf.h

```diff
@@ -167,6 +167,26 @@ struct bt_hfp_hf_cb {
 	 */
 	void (*clip)(struct bt_conn *conn, char *number, uint8_t type);
 #endif /* CONFIG_BT_HFP_HF_CLI */
+#if defined(CONFIG_BT_HFP_HF_VOLUME)
+	/** HF microphone gain notification callback to application
```

Comment:
Since you have the #ifdef blocks, none of the doxygen blocks inside it will actually get used, unless you add the respective KConfig options to `PREDEFINED` in `doc/zephyr.doxyfile.in`.

Reply:
Updated